src/components/Checkbox/BaseCheckbox.js

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
 import {View, Pressable} from 'react-native';
 import PropTypes from 'prop-types';
-import styles from '../styles/styles';
-import themeColors from '../styles/themes/default';
-import stylePropTypes from '../styles/stylePropTypes';
-import Icon from './Icon';
-import * as Expensicons from './Icon/Expensicons';
+import styles from '../../styles/styles';
+import themeColors from '../../styles/themes/default';
+import stylePropTypes from '../../styles/stylePropTypes';
+import Icon from '../Icon';
+import * as Expensicons from '../Icon/Expensicons';
+
+const requiredPropsCheck = (props, componentName) => {
+    if (!props.onMouseDown && !props.onPress) {
+        return;
+    }
+    return new Error(`One of "onMouseDown" or "onPress" must be provided in ${componentName}`);
```

Comment:
![Simulator Screen Shot - iPhone 12 - 2022-11-30 at 20 47 00](https://user-images.githubusercontent.com/9059945/204882675-802a20e3-4d47-47b3-9b85-8ed44acf783d.png)

There is propTypes error on application start

Reply:
Android?

Reply:
And does it work despite the warning?

Reply:
> Android also or iOS only?

All of the platforms

Reply:
> And does it work despite the warning?

No, take a look at [this](https://github.com/Expensify/App/pull/13187#discussion_r1036329464) comment

please test everything before opening a PR

Reply:
Actually, when I did the change during the PR, that is when this bug was introduced. I tested it thoroughly before submitting. In the requiredPropCheck function, the conditions were negated by mistake which caused it to show the error even when the props were set. But anyway, I fixed it and also changed the position of the props spreading to be more reliable. Do you want me to send the videos for any platform again?

Reply:
> Do you want me to send the videos for any platform again?

No, just verify that it is working